web/steps.php

```diff
@@ -23,6 +23,23 @@
 
 function petition_form_intro() {
     global $site_name;
+    // Get a nice string to print for the maximum deadline from the cobrand if
+    // one's setup
+    $deadline_limits = cobrand_creation_deadline_limit();
+    $maximum = 'for up to 12 months';
+    if (array_key_exists('date', $deadline_limits)) {
```

Comment:
It'd be more consistent with the conditionals below and also exclude falsy values if you used `if ($deadline_limits['date'])` here.